modules/solid_mechanics/doc/content/source/vectorpostprocessors/AverageSectionValueSampler.md

```diff
@@ -27,14 +27,29 @@ This postprocessor should only be used for regularly meshed geometries as it rel
 same weights on the cross section. There is no check within the code for this condition, so the user must
 ensure that the mesh is reasonably uniform.
 
+Because this VectorPostprocessor is intended to be applied to geometries with extruded meshes, with layers
+of nodes that all occupy the same axial position, it will generate an error if the numbers of nodes located
+at the various axial positions vary. However, there are valid scenarios when this is not the case. For example,
+if part of the extruded mesh is refined, the number of nodes per layer will differ along the axis of the
+extrueded section. The [!param](/VectorPostprocessors/AverageSectionValueSampler/require_equal_node_counts) parameter can be set to `false` in such situations to skip
+that check. The number of nodes per position is reported in the ouputs as the `node_count` vector to allow the
+user to ensure that each layer contains all expected nodes.
+
+The optional [!param](/VectorPostprocessors/AverageSectionValueSampler/symmetry_plane) parameter is intended to be used when there is a symmetry plane passing through
+the extruded section being evaluated. If there is volumetric expansion in the section and this is used to 
+compute displacements, the results will be skewed because only the expansion on one side of the symmetry plane
+is considered, when it should be balanced out by an expansion in the opposite direction on the other side of
+that symmetry plane. To account for this, if [!param](/VectorPostprocessors/AverageSectionValueSampler/symmetry_plane) is specified, the set of variables is treated as a size-three vector, and the component of that vector in the direction of the specified symmetry plane is removed. The [!param](/VectorPostprocessors/AverageSectionValueSampler/symmetry_plane) parameter defines the normal to the symmetry plane, and when this option is used, a set of
```

Comment:
so you only support 3 variables then?

or we can do 3 variables for the positions, then however many for the sampling

Reply:
The variables are not used to define positions. If you use the symmetry plane option, a set of 3 variables needs to be provided to make up the components of a vector in 3D space.

Reply:
ok I think I get it now. it does not matter

Reply:
but can you add variables after the 3 variables making the component? or are you only allowed 3 variables?

Reply:
Only 3 would be allowed